OpenRA.Mods.Common/UpdateRules/Rules/20171014/ReplaceRequiresPower.cs

```diff
@@ -0,0 +1,43 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class ReplaceRequiresPower : UpdateRule
+	{
+		public override string Name { get { return "Replace 'RequiresPower' with 'GrantConditionOnPowerState'"; } }
+		public override string Description
+		{
+			get
+			{
+				return "'RequiresPower' has been replaced with 'GrantConditionOnPowerState' which\n" +
+					"toggles a condition depending on the power state.\n Possible PowerStates are: Normal " +
+					"(0 or positive), Low (negative but higher than 50% of required power)\nand Critical (below Low).";
```

Comment:
Would look better if the newline was moved from before the "and" to before the "50%"